### DISCUSSION TITLE
Old Gaggia Classic - Valve pressure question

### DISCUSSION THREAD

**Discussion by shipcommit** [Zer0-bit/gaggiuino#387] [Q&A] [answered]
What pressure do I need to use for this mod?

My 8 year old Gaggia Classic uses 6.5 bars, which is adjusted with a screw.

**Comment by Zer0-bit** [comment]
All of it.

**Reply by shipcommit** [reply]
What do you mean?

**Reply by Zer0-bit** [reply, marked as answer]
Max pressure you ever want to achieve + 2 bar.
